hr_employee_document/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Employee Document",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
+    "author": "Brainbean Apps, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": "Documents attached to the employee profile",
+    "depends": ["hr", "attachment_indexation"],
```

Comment:
Dependency with `attachment_indexation` is not necessary: with this module the attachments are already accessible by smart button.

Reply:
Edited.